mail-client/geary/geary-3.34.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+VALA_MIN_API_VERSION="0.40"
```

Comment:
I think this needs to be VALA_MIN_API_VERSION="0.44"

Build fails with dev-lang/vala-0.42.7:
```
../geary-3.34.1/src/client/components/client-web-view.vala:310.9-310.38: error: The name `enable_javascript_markup' does not exist in the context of `WebKit.Settings?'
        setts.enable_javascript_markup = false;
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
enable_javascript_markup needs webkit 2.24 support in Vala
https://github.com/GNOME/vala/blob/ff57dee633975cba39883af7caf4d21e4909402e/vapi/webkit2gtk-4.0.vapi#L653
First added in Vala 0.44.0
https://github.com/GNOME/vala/commit/8a2e9e0f01b82bd81ab9ddf895733c6bf551fccc
https://github.com/GNOME/vala/blob/f413cb32470d9c2598416772bc0166c6f974ec9f/NEWS#L305